common/modules/disable-add-liq/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![no_std]
+
+multiversx_sc::imports!();
+
+pub const ADD_LIQ_ENABLED: bool = false;
+pub const ADD_LIQ_DISABLED: bool = true;
+
+#[multiversx_sc::module]
+pub trait DisableAddLiqModule {
+    #[only_owner]
+    #[endpoint(enableAddLiq)]
+    fn enable_add_liq(&self) {
+        self.add_liq_disabled().set(ADD_LIQ_ENABLED);
+    }
+
+    #[only_owner]
+    #[endpoint(disableAddLiq)]
+    fn disable_add_liq(&self) {
+        self.add_liq_disabled().set(ADD_LIQ_DISABLED);
+    }
+
+    fn require_add_liq_enabled(&self) {
+        require!(
+            self.add_liq_disabled().get() == ADD_LIQ_ENABLED,
+            "Add Liquidity is disabled"
+        );
+    }
+
+    #[view(isAddLiqDisabled)]
+    #[storage_mapper("addLiqDisabled")]
+    fn add_liq_disabled(&self) -> SingleValueMapper<bool>;
```

Comment:
I would rename this from `add_liq_disabled` to `add_liq_status` which will be set to `ADD_LIQ_DISABLED` or `ADD_LIQ_ENABLED`.

Reply:
Nah, then I'd have to specifically set it to `true` to have it enabled. I figured it's easier this way.